serverless/functions/serverless-functions-overview.adoc

```diff
@@ -0,0 +1,44 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-functions-overview"]
+= {FunctionsProductShortName} overview
+:context: serverless-functions-overview
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+:FeatureName: {FunctionsProductName}
+include::modules/technology-preview.adoc[leveloffset=+2]
+
+{FunctionsProductName} enables developers to create and deploy stateless, event-driven functions as a Knative service on {product-title}.
+
+The `kn func` CLI is provided as a plugin for the Knative `kn` CLI. {FunctionsProductName} uses the link:https://buildpacks.io/[CNCF Buildpack API] to create container images. Once a container image has been created, you can use the `kn func` CLI to deploy the container image as a Knative service on the cluster.
+
+[id="serverless-functions-overview-runtimes"]
+== Supported runtimes
+
+{FunctionsProductName} provides templates that can be used to create basic functions for the following runtimes:
+
+// add xref links to docs once added
+* Node.js
+* Python
+* Golang
+* Quarkus
+//* SpringBoot - TBC
+
+[id="serverless-functions-overview-setup"]
+== Setting up {FunctionsProductShortName}
+
+To use {FunctionsProductName}, you must install the xref:../../serverless/installing-kn.adoc#installing-kn[`kn` CLI]. Installing the `kn` CLI enables the use of the `kn func` commands used to create and manage functions.
+
+{FunctionsProductName} also requires the use of Docker Container Engine or podman, and access to a publicly available image registry.
```

Comment:
@jrangelramos what was the problem with Podman? It should work

Reply:
So far docker is the only option. CLI attempt to build thru docker dameon directly.
https://issues.redhat.com/browse/SRVOCF-282

Sample output when building without docker:
```
Building function image
Error: failed to fetch builder image 'quay.io/boson/faas-nodejs-builder:latest': Cannot connect to the Docker daemon at unix:///run/user/1000/docker.sock. Is the docker daemon running?
output: 2021/04/29 10:08:22.494032 DEBUG:  Pulling image quay.io/boson/faas-nodejs-builder:latest
 ```

Reply:
This is prime information for docs and qe! :) 

I believe this is what you should do to use podman - @matejvasek can confirm

```console
$ podman system service --time=0 tcp:0.0.0.0:1234 & # let run in background or another terminal
$ export DOCKER_HOST=tcp://127.0.0.1:1234
$ func build -v
```

Reply:
yes.. it worked ok!

Reply:
@lance might be a basic question, but what do those commands do, i.e. why does someone need to run them? I've added them already as their own module for podman, but I think some extra context for the steps would be nice.

Reply:
@abrennan89 if a user does not have Docker installed, or chooses not to use it, we support podman - as you know. This means that, instead of the Docker daemon running on the system, there is a podman service that is capable of responding to the same requests as Docker. This is possible because podman supports the Docker API. In the commands above, the first command starts podman as a service on the local computer, listening on port 1234. The second command establishes an environment variable that the func internals use to determine how/where to connect when building the project. Does this help?

Reply:
Thank you! Made some updates to explain the commands.